include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -133,6 +133,14 @@ typedef enum OrtErrorCode {
   ORT_EP_FAIL,
 } OrtErrorCode;
 
+// use -1 to allow ORT to choose good defaults
+typedef struct OrtArenaCfg {
+  int max_mem;
+  int arena_extend_strategy;  // 0 = kNextPowerOfTwo, 1 = kSameAsRequested // TODO can be an enum
+  int initial_chunk_size_bytes;
+  int max_dead_bytes_per_chunk;
+} OrtArenaCfg;
```

Comment:
These probably need a lot more explanation of what they do and how to choose appropriate values.

Reply:
Will add it in a separate PR in the tuning document.